src/PrestaShopBundle/Controller/Api/TranslationController.php

```diff
@@ -315,9 +315,9 @@ private function getNormalTree($lang, $type, $selected, $search = null)
      */
     private function getModulesTree($lang, $type, $selected, $search = null)
     {
-        $moduleProvider = $this->container->get('prestashop.translation.module_provider');
-        $moduleProvider->setModuleName($selected);
 
+        $moduleProvider = $this->container->get('prestashop.translation.external_module_provider');
```

Comment:
this is a POC: in the end, the original module provider needs to use external module provider if it doesn't find a Catalogue.